static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -319,7 +319,8 @@ class ProjectPerformance extends AsyncView<Props, State> {
     return (
       <Fragment>
         <SettingsPageHeader title={t('Performance')} />
-        <PermissionAlert access={requiredScopes} />
+        <PermissionAlert access={requiredScopes} project={project} />
```

Comment:
```suggestion
        <PermissionAlert project={project} />
```
the default `access` in PermissionAlert is `['project:write']` and `requiredScopes = ['project:write']`